pymagicc/io.py

```diff
@@ -2095,3 +2119,106 @@ def get_generic_rcp_name(inname):
     except KeyError:
         error_msg = "No generic name for input: {}".format(inname)
         raise ValueError(error_msg)
+
+
+# TODO: move out of here in clean up
+def join_timeseries(base, overwrite, join_linear=None):
```

Comment:
`join_linear` - should this be a tuple (while the code would still accept a list) or rather, `start`, `stop` - i think we usually have start and end points as separate args (correct me if i'm wrong or if there is a specific reason to have a list)

Reply:
I think having separate start and end is a bit dangerous as if you have a start arg but no end, I'm not sure what should happen... Tuple is good, have altered